src/fhirtypes/common.ts

```diff
@@ -979,6 +979,16 @@ export function replaceReferences<T extends AssignmentRule | CaretValueRule>(
         clone = cloneDeep(rule);
         (clone.value as FshReference).reference = value.reference.slice(0, firstPipe);
         clone = replaceReferences(clone, tank, fisher);
+      } else if (type == null && id == null && value.reference) {
+        // if the reference to an entity is provided but is unable to be resolved
+        if (!value.reference.startsWith('urn:')) {
+          if (!value.reference.includes('/')) {
```

Comment:
The way this logic is written with the nested if statements makes me think there's something else we want to do if the value starts with `urn:` but does contain a `/`. I think it would be simpler to just have one if statement with both conditions combined with "and" (i.e. `!value.reference.startsWith('urn:') && !value.reference.includes('/')`). While that doesn't change the logic as it currently is, I think it makes the intent a little clearer.

Reply:
Or there is, of course, the equivalent inverse expression: `!(value.reference.startsWith('urn:') || value.reference.includes('/'))`. Take your pick!

Reply:
Oh right. That's his name!